insights/parsers/freeradius.py

```diff
@@ -0,0 +1,53 @@
+"""
+FreeRadius - Freeradius configuration files
+===========================================
+
+FreeRadius is a parser for the freeradius configuration files usually in
+``/etc/raddb``.
+Files ending with ``.rpmnew`` are excluded because they are backups made when
+ugprading but customer modified the configuration files, so are not used by the
+service.
+
+Because this parser reads multiple files, the files are stored as a list within
+the parser and need to be iterated through in order to find specific
+configuration files.
+
+Examples:
+
+    >>> for cfgfile in shared[FreeRadius]: # Parser contains list of all configuration files
+    ...     print "Path:", cfgfile.file_path
+    ...
+    Path: etc/raddb/radiusd.conf
+    Path: etc/raddb/proxy.conf
+    Path: etc/raddb/policy.d/filter
+
+"""
+
+from .. import Parser, parser
+from . import SkipException
+
+
+@parser("freeradius")
```

Comment:
Parsers in 3.x depend directly on their entry from the `Specs` class in `insights/specs/__init__.py`.